File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/state/LiveState.java
package com.trtc.uikit.livekit.state;

import com.trtc.uikit.livekit.state.view.ViewState;
import com.trtc.uikit.livekit.state.operation.OperationState;

public class LiveState {
    public OperationState operationState = new OperationState();
    public ViewState      viewState      = new ViewState();

    public void reset() {
        viewState.reset();
        operationState.reset();
    }
}

File: Flutter/lib/state/live_state.dart
import 'package:tencent_live_uikit/state/operation/operation_state.dart';
import 'package:tencent_live_uikit/state/view/view_state.dart';

class LiveState {
  final OperationState operationState = new OperationState();
  final ViewState      viewState      = new ViewState();

  void reset() {
    viewState.reset();
    operationState.reset();
  }
}